File: app/src/main/java/com/arny/lubereckiy/models/Sticker.java

package com.arny.lubereckiy.models;

import com.arny.arnylib.utils.Utility;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Sticker {

	@Override
	public String toString() {
		return Utility.getFields(this);
	}

	@SerializedName("text")
    @Expose
    private String text;
    @SerializedName("color")
    @Expose
    private String color;

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }

}

File: app/src/main/java/com/arny/pik/data/models/Sticker.kt
package com.arny.pik.data.models

import com.arny.pik.utils.Utility
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Sticker {

    @SerializedName("text")
    @Expose
    var text: String? = null
    @SerializedName("color")
    @Expose
    var color: String? = null

    override fun toString(): String {
        return Utility.getFields(this)
    }

}
